sccp/imports/scRNA.py

```diff
@@ -55,7 +55,7 @@ def import_thompson_drug() -> anndata.AnnData:
     return data
 
 
-def ThompsonXA_SCGenes(offset: float = 1.0) -> anndata.AnnData:
+def ThompsonXA_SCGenes(annData = False, offset: float = 1.0) -> anndata.AnnData:
```

Comment:
Why are you messing with this function? I wrote a test with data from the Theis lab.